docs/visual-basic/programming-guide/language-features/procedures/operator-procedures.md

```diff
@@ -37,13 +37,15 @@ An operator procedure is a series of Visual Basic statements that define the beh
   
 ## Declaration Syntax  
  The syntax for declaring an operator procedure is as follows:  
+ 
+ ```vb 
+ Public Shared [Widening | Narrowing] Operator *operatorsymbol* ( *operand1* [,  *operand2* ]) As *datatype*  
```

Comment:
The \* here were used to make the names italic. By using a code block, we should probably remove those:

```suggestion
 Public Shared [Widening | Narrowing] Operator operatorsymbol ( operand1 [,  operand2 ]) As datatype  
```

This is how the change renders right now:
![image](https://user-images.githubusercontent.com/12971179/59458101-9f7c1780-8dce-11e9-8c14-7270e19f1e7f.png)